src/routes/apiRouter.js

```diff
@@ -1,10 +1,14 @@
 import { Router } from 'express';
 import handleResponse from '../controllers/handleResponse.js';
 import userRouter from './userRouter.js';
+import loginRouter from './loginRouter.js';
+import { refresh } from '../controllers/refreshToken.controller.js';
 
 const apiRouter = Router();
 
 apiRouter.use('/users', userRouter);
+apiRouter.use('/login', loginRouter);
```

Comment:
Can we change this route to something else. As a suggestion. 

> Reason

If we change the resource to **`/acounts/`** then we can easily moves other routes like **`/acounts/password-reset`** ,**`/password-change`** there. then we can have **`/accounts/login`**

What other guyz think @Abdulkeza @Yvad60